apps/mixpost/metadata/description.md

```
@@ -1,13 +1,14 @@
-An admin user will be created automatically. SSH into your Tipi server and run `docker logs mixpost 2>&1 | grep "Your password" ` to find out the password.
+An admin user will be created automatically. 
+Initial startup may take a few minutes, you should see the default admin crendential when the app is ready.
 
 You can log in to Mixpost at /mixpost using the admin user account created.
 
-## Inital User
+### Initial User
 
-Username: admin@admin.com
-Password: SSH into your Tipi server and run `docker logs mixpost 2>&1 | grep "Your password" ` 
+- Email: admin@example.com
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)